plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -39,36 +39,14 @@ protected function get_block_type_uses_context() {
 	}
 
 	/**
-	 * Get stock text based on stock. For example:
-	 * - In stock
-	 * - Out of stock
-	 * - Available on backorder
-	 * - 2 left in stock
+	 * Get the product types that are allowed in the editor.
```

Comment:
Should we update this comment? It's not 100% clear to me what the "in the editor" means here. :thinking: